src/site/content/en/progressive-web-apps/what-are-pwas/index.md

```diff
@@ -0,0 +1,135 @@
+like take pictures, see playing songs listed on the home screen, or control song
+playback while in another app. Native applications feel like _part_ of the
+device they run on.
+
+<figure class="w-figure w-figure--center">
+  <img src="./capabilities-reach.svg" style="max-width: 100%;"
+    alt="A graph illustrating the relative capabilites and reach of native apps, with high capabilities, web apps, with high reach, and progressive web apps, which have both high capabilities and high reach."/>
+  <figcaption class="w-figcaption w-figcaption--fullbleed">
+    Capabilities vs reach of native apps, web app, and progressive web apps.
+  </figcaption>
+</figure>
+
+If you think about native apps and web apps in terms of capabilities and reach,
+native apps represent the best of capabilities whereas web apps represent the
+best of reach. So where do Progressive Web Apps fit in?
+
+Progressive Web Apps (PWA) are built and enhanced with modern APIs to deliver
+native-like capabilities, reliability, and installability while reaching
+_anyone, anywhere, on any device_ with a single codebase.
+
+## The three app pillars
+
+Progressive Web Apps are web applications that have been designed so they are
+capable, reliable, and installable. These three pillars transform them into an
+experience that feels like a native application.
+
+### Capable
+
+The web is quite capable in its own right today. You can build a hyper-local
+video chat app using WebRTC, geolocation, and push notifications. You can make
+that app installable and take those conversations virtual with WebGL and WebVR.
+With the introduction of Web Assembly, developers can tap into other ecosystems,
+like C, C++, and Rust, and bring decades of work and capabilities to the web
+too! [Squoosh.app](https://squoosh.app/), for instance, leveraged this for its
+advanced image compression.
+
+Until recently, only native apps could really lay claim to these capabilities.
+While some capabilities are still out of the web's reach, new and upcoming APIs
+are looking to change that, expanding what the web can do with features like
+file system access, media controls, app badging, and full clipboard support. All
+of these capabilities are built with the web's secure, user-centric permission
+model, ensuring that going to a website is never a scary proposition for users.
+
+Between modern APIs, Web Assembly, and new and upcoming APIs, web applications
+are more capable than ever, and those capabilities are only growing.
+
+### Reliable
+
+A reliable Progressive Web App feels fast and dependable regardless of the
+network.
+
+Speed is critical for getting users to _use_ your experience. In fact, as page
+load times go from 1 second to ten seconds, the probability of a user bouncing
+[increases by 123%](https://www.thinkwithgoogle.com/marketing-resources/data-measurement/mobile-page-speed-new-industry-benchmarks/).
+Performance doesn't stop `onload`. Users should never wonder whether their
```

Comment:
Performance doesn't stop after the `onload` event.

I know I usually say to remove words. Sometimes it's needed for clarity.

Reply:
```suggestion
Performance doesn't stop after the `onload` event. Users should never wonder whether their
```